### PR TITLE
Feature/footer-update (Faldó)

### PR DESCRIPTION
**Pull request by uriel-huertas**
#### :tophat: What? Why?
This PR adds Decidim's footer to Participa. 

### :camera: Screenshots (optional)
image

### CODE REVIEW

File: decidim-home/app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
-		<div class="medium-10 small-12 column">
-			<div class="part-justify part-avis-legal">
-				<a href="http://web.gencat.cat/<%= I18n.locale %>/menu-ajuda/ajuda/avis_legal/" target="_blank">
-						<%= t("static.footer.legal_advice_title")%>:
-				</a>
-				<%= t("static.footer.legal_advice_text")%>
-			</div>
-		</div>
-	</div>
+  <div class="row">
```
Comment (tramuntanal) [inline decidim-home/app/views/layouts/_footer.html.erb]: Can you rename `layouts/_footer.html.erb` to `layouts/_main_footer.html.erb' and add the new content into a new partial `layouts/_mini_footer.html.erb` invoked below?

File: app/assets/images/decidim-logo.svg [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (tramuntanal) [inline app/assets/images/decidim-logo.svg]: The logo and the badge are already in decidim, I guess we don't need to add them here again. Can you confirm @isaacmg410 ?

File: db/dev-user.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+email = "admin@example.org"
```
[the remark this answers is not in the document]
Comment (uriel-huertas) [reply db/dev-user.rb]: This file adds the admin user for the docker environment. 
For some reason, rake's db:seed task wasn't working inside the docker container and this acts as a workarround due that I had no time to debug it.

After this explanation, if you still want me to remove it, i'll do so.

File: Gemfile [hunk 4]
```diff
@@ -29,6 +29,8 @@ gem "devise", ">= 4.7.1"
 gem "nokogiri", ">= 1.10.4"
 # gem sprockets in version 4.0 breaks Decidim.Temporal fix at 10/10/2019
 gem "sprockets", "~> 3.7.2"
+# Temporal fix for: https://github.com/decidim/decidim/issues/5257
```
[the remark this answers is not in the document]
Comment (uriel-huertas) [reply Gemfile]: Done https://github.com/gencat/participa/pull/108/commits/278db46102135c9510be05b8aea0ca25ee75be5e

File: decidim-home/app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (uriel-huertas) [reply decidim-home/app/views/layouts/_footer.html.erb]: Done https://github.com/gencat/participa/pull/108/commits/7caa7c90b1b7160853146fd90ff41583b67f20e4

File: db/dev-user.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (tramuntanal) [reply db/dev-user.rb]: Yes, remove please. Thanks

File: decidim-home/app/views/layouts/_mini_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    </div>
+    <div class="medium-3 large-2 column">
+      <div class="decidim-logo">
+        <a rel="decidim" href="https://decidim.org/" target="_blank">
+          <%= image_tag("decidim-logo.svg") %>
+        </a>
+      </div>
+    </div>
+  </div>
+</div>
```
[the remark this answers is not in the document]
Comment (tramuntanal) [reply decidim-home/app/views/layouts/_mini_footer.html.erb]: @uriel-huertas  I thought it was copied...

File: db/dev-user.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (shad0wflame) [reply db/dev-user.rb]: Done https://github.com/gencat/participa/pull/108/commits/f8c269d54540d5f4ae76612d457834d129dee0db

File: app/assets/images/decidim-logo.svg (hunk 2 above)
Comment (shad0wflame) [reply app/assets/images/decidim-logo.svg]: Done https://github.com/gencat/participa/pull/108/commits/f8c269d54540d5f4ae76612d457834d129dee0db

Comment (uriel-huertas) [reply app/assets/images/decidim-logo.svg]: Done https://github.com/gencat/participa/pull/108/commits/f8c269d54540d5f4ae76612d457834d129dee0db

File: db/dev-user.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (uriel-huertas) [reply db/dev-user.rb]: Done https://github.com/gencat/participa/pull/108/commits/f8c269d54540d5f4ae76612d457834d129dee0db

File: decidim-home/app/views/layouts/_mini_footer.html.erb (hunk 5 above)
[the remark this answers is not in the document]
Comment (uriel-huertas) [reply decidim-home/app/views/layouts/_mini_footer.html.erb]: Done https://github.com/gencat/participa/pull/108/commits/317caea548d71b19e509e30d0fffde8de03916df

File: decidim-home/app/views/layouts/_mini_footer.html.erb [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<div class="mini-footer">
+  <div class="row">
+    <div class="medium-3 large-4 column">
+      <a rel="license" class="cc-badge" href="http://creativecommons.org/licenses/by-sa/4.0/" target="_blank">
+        <%= image_tag("cc-badge.png") %>
+      </a>
+      Web creada amb <a target="_blank" href="https://github.com/decidim/decidim">programari lliure</a>.
```
[the remark this answers is not in the document]
Comment (uriel-huertas) [reply decidim-home/app/views/layouts/_mini_footer.html.erb]: Resolved in this thread https://github.com/gencat/participa/pull/108/files/72975f753900696a587a6637f54011d850e49135#diff-97df281b8cda77cd6ebdaed05cf0dac4

**Pull request merged**